etc/config-template.toml

```diff
@@ -769,8 +769,8 @@
 # wait-for-lock-timeout = 3000
 
 ## Time to delay in milliseconds before responding to TiDB when other transactions
```

Comment:
I think this description needs some updating... Change to `The maximum delay in milliseconds before...`